### PR TITLE
Library landing page bottom cta

### PR DESCRIPTION
**Pull request by yipingliao**
@2is10 
https://app.asana.com/0/19898402862947/25955788453838/f

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/libraries/library.styl [hunk 1]
```diff
@@ -16,7 +16,7 @@ rel-libs-width = 210px
   vertical-align: top
 
 .kf-lib-footer
-  margin-bottom: 30px
+  padding-bottom: 30px
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/libraries/library.styl:19]: Apparently Safari doesn't honor this margin when there is no other element under it. Switching to a padding instead.

File: kifi-backend/modules/shoebox/angular/src/common/directives/pageBottomCTA/pageBottomCta.styl [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.kf-page-bottom-cta
+  position: fixed
+  bottom: 0
+  margin: 0 auto
+  width: 100%
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/common/directives/pageBottomCTA/pageBottomCta.styl:5]: does `margin: 0 auto` accomplish anything here?

File: kifi-backend/modules/shoebox/angular/src/common/directives/pageBottomCTA/pageBottomCta.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          $rootScope.$emit('trackLibraryEvent', 'click', {
+            action: 'clickedLoginPopup'
+          });
+        };
+
+        function onScroll() {
+          if (!scope.show) {
+            $rootScope.$emit('trackLibraryEvent', 'view', { type: 'libraryLandingPopup' });
+            scope.show = true;
+            angular.element('.kf-lib-footer').css('padding-bottom', '300px');
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/common/directives/pageBottomCTA/pageBottomCta.js:37]: If this thing came just before `.kf-lib-footer` in the DOM and used `ng-if`, then this extra padding could be accomplished in the stylesheet.

``` less
.kf-lib-footer
  padding-bottom: 30px

  .kf-pbc+&
    padding-bottom: 300px
```

Not critical, just an idea.

**Comment by 2is10** [comment]
:+1: nicely done

**Pull request merged**